src/main/java/game/map/MapHelper.java

```diff
@@ -8,12 +8,18 @@ public class MapHelper {
     public static Continent getContinentWithId(Map map, int id) {
 
         return map.getContinents()
-                .stream().filter(continent -> continent.getId() == id).toList().get(0);
+                .stream()
+                .filter(continent -> continent.getId() == id)
+                .findFirst()
+                .orElse(null);
     }
 
     public static Country getCountryWithId(Map map, int id) {
 
         return map.getCountries()
-                .stream().filter(country -> country.getId() == id).toList().get(0);
```

Comment:
Thank you for updating these.

Whoever is calling this method needs to add a null check to avoid `NullPointerException`s. 
I have used called these methods in the `MapLoader` class anyway, so I can add this null check at all places.